resources/lang/vi/authorization.php

```diff
@@ -21,134 +21,134 @@
 return [
     'beatmap_discussion' => [
         'destroy' => [
-            'is_hype' => 'Can not undo hyping.',
-            'has_reply' => 'Can not delete discussion with replies',
+            'is_hype' => 'Không thể hoản lại hype.',
+            'has_reply' => 'Không thể xóa cuộc thảo luận cùng trả lời',
         ],
         'nominate' => [
-            'exhausted' => 'You have reached your nomination limit for the day, please try again tomorrow.',
+            'exhausted' => 'Bạn đã đạt mốc giới hạn đề cử (nominate) của hôm nay, hảy thử lại vào ngày mai.',
         ],
         'resolve' => [
-            'not_owner' => 'Only thread starter and beatmap owner can resolve a discussion.',
+            'not_owner' => 'Chỉ có người bắt đầu thread và chủ beatmap mới có thể giải quyết cuộc thảo luận.',
         ],
 
         'vote' => [
-            'limit_exceeded' => 'Please wait a while before casting more votes',
-            'owner' => "Can't vote on own discussion.",
-            'wrong_beatmapset_state' => 'Can only vote on discussions of pending beatmaps.',
+            'limit_exceeded' => 'Vui lòng đợi một lúc trước khi bình chọn thêm',
+            'owner' => 'Không thể bình chọn cuộc thảo luận của bạn.',
+            'wrong_beatmapset_state' => 'Chỉ có thể bình chọn cuộc thảo luận của beatmap đang chờ.',
         ],
     ],
 
     'beatmap_discussion_post' => [
         'edit' => [
-            'system_generated' => 'Automatically generated post can not be edited.',
-            'not_owner' => 'Only the poster can edit post.',
+            'system_generated' => 'Không thể chỉnh sửa bài đăng được tạo tự động.',
+            'not_owner' => 'Chỉ người đăng mới có thể chỉnh sửa bài đăng.',
         ],
     ],
 
     'chat' => [
         'channel' => [
             'read' => [
-                'no_access' => 'Access to requested channel is not permitted.',
+                'no_access' => 'Không được phép truy cập vào kênh đã yêu cầu.',
```

Comment:
Robot translation
Cứ dịch đơn giản như dịch "access denied" là cũng ok rồi ("no_access")

Reply:
🤔

Reply:
not really robot tho 🤔